packages/tools/storybook/stories/components/form-field/form-field.web-components.stories.ts

```diff
@@ -0,0 +1,90 @@
+import { html } from 'lit-html';
+import { iframe } from '../../../.storybook/iframe';
+import { odsFormFieldDefaultAttributes } from '@ovhcloud/ods-core';
+import { defineCustomElements } from '@ovhcloud/ods-stencil-form-field/loader';
+import {
+  extractArgTypes,
+  extractStoryParams,
+  getTagAttributes,
+} from '../../../core/componentHTMLUtils';
+
+import changelog from '@ovhcloud/ods-stencil-form-field/CHANGELOG.md';
+import page from './form-field.web-component.stories.page.mdx';
+
+defineCustomElements();
+
+/* Default story parameters  */
+const storyParams = {
+  flex: {
+    category: 'General',
+    defaultValue: odsFormFieldDefaultAttributes.flex,
+  },
+  error: {
+    category: 'General',
+    defaultValue: odsFormFieldDefaultAttributes.error,
+  },
+  label: {
+    category: 'Misc',
+    defaultValue: true,
+    control: {
+      type: 'boolean',
```

Comment:
For label, visualHint, input and helper, could you please use category 'Slot' instead of 'Misc' (and no boolean control) so, our users can easily play in the sandbox? (by adding an osds-tooltip instead of an osds-text in visualHint for example etc.)
You can check in Button story (buttonContent) for instance

Reply:
Oh very nice! Didn't know we could do that